docs/reference_guides/model_libraries/models_extra/phe.rst

```diff
@@ -0,0 +1,172 @@
+
+.. figure:: ./phe_a.png
+  :alt: Z-configuration Plate Heat Exchanger with P passes
+  :width: 90%
+  :align: center
+
+  **Figure 1(A). Z-configuration Plate Heat Exchanger with P passes**
+
+.. figure:: ./phe_b.png
+  :alt: Basic details of a Chevron Plate
+  :width: 100%
+  :align: center
+
+  **Figure 1(B). Basic details of a Chevron Plate**
+
+
+Degrees of Freedom
+------------------
+
+The Plate Heat Exchanger model has 3 design parameters and 7 design variables.
+
+Design Parameters and Variables:
+
+    * number of passes (mutable parameter),
+    * channels per pass (mutable parameter),
+    * number of divider plates (mutable parameter,
+    * plate length,
+    * plate width,
+    * plate thickness,
+    * port diameter,
+    * plate thermal conductivity, and
+    * total heat transfer area.
+
+Model Structure
+---------------
+
+The Plate Heat Exchanger unit model builds off the core
+:ref:`HeatExchangerNTU model <reference_guides/model_libraries/generic/unit_models/heat_exchanger_ntu:Heat Exchanger using the NTU Method>`,
+and heat transfer is based on the Effectiveness-Number of Transfer Units
+(e-NTU method).
+
+Parameters
+----------
+
+The following parameters can be set via configuration arguments when instantiating the Plate Heat Exchanger model, or modified later.
+
+=========================== ==================== =========== =============================================================================
+Variable                    Symbol               Index Sets  Doc
+=========================== ==================== =========== =============================================================================
+number_of_passes            :math:`N_{passes}`   None        Number of passes in heat exchanger unit
+channels_per_pass           :math:`N_{channels}` None        Number of channels per heat exchanger pass (assumed equal in all plates)
+number_of_divider_plates    :math:`N_{dividers}` None        Number of divider plates in heat exchanger assembly
+=========================== ==================== =========== =============================================================================
+
+Variables
+---------
+
+The following variables are declared in addition to those variables created by the HeatExchangerNTU class.
+
+=========================== ================== =========== =============================================================================
+Variable                    Symbol             Index Sets  Doc
+=========================== ================== =========== =============================================================================
+plate_length                :math:`L`          None        Length of a heat exchanger plate
+plate_width                 :math:`W`          None        Width of a heat exchanger plate
+plate_thickness             :math:`H`          None        Thickness of a heat exchanger plate
+plate_pact_length           :math:`L_{pact}`   None        Compressed plate pact length
```

Comment:
This is probably just my ignorance, but what's "pact?"

Reply:
I am not certain myself, but I believe it is a technical term - there is an image in the docs which illustrates it.

Reply:
Ah ok, I confess I didn't look at the images.